app/assets/stylesheets/forever_style_guide/globals/_colors.scss

```diff
@@ -44,6 +44,11 @@ $color-danger:          colorify('danger', $color-forever_red) !default;
 $color-background:      set_color('background', $color-forever_beige) !default;
 $color-text:            set_color('text', $color-forever_gray) !default;
 
+// Forever LIVE! 2016 brand colors
+$color-live16-green: colorify('live16-green', #367860) !default;
```

Comment:
On the whole the indentation is inconsistent in this file. I think the uniform indent ala the grays and semantic colors is easiest to read for this type of stylesheet. Do you mind making this consistent across colors.scss?